BitwardenShared/UI/Vault/VaultItem/AddEditItem/AddEditLoginItem/AddEditLoginItemView.swift

```diff
@@ -47,7 +47,14 @@ struct AddEditLoginItemView: View {
                     fido2Credential.creationDate.formatted(date: .numeric, time: .omitted),
                     fido2Credential.creationDate.formatted(date: .omitted, time: .shortened)
                 )
-            )
+            ) {
+                if store.state.canViewPassword {
+                    AccessoryButton(asset: Asset.Images.minusRound, accessibilityLabel: Localizations.removePasskey) {
+                        store.send(.removePasskeyPressed)
```

Comment:
🤔 Maybe I'm missing it but could we add a snapshot test for when `canViewPassword` is `false`? Also Codecov is missing a test for `store.send(.removePasskeyPressed)`.

Reply:
@fedemkr Can you check if this snapshot is the one you are looking for [test_snapshot_edit_full_disabledViewPassword.1.png](https://github.com/bitwarden/ios/blob/pm-5885/allow-passkey-deletion-edit-view/BitwardenShared/UI/Vault/VaultItem/AddEditItem/__Snapshots__/AddEditItemViewTests/test_snapshot_edit_full_disabledViewPassword.1.png)

Reply:
Good callout on the Codecov, ty 💯

Reply:
Yep, that was the one I was looking for, thanks!